spec/controllers/api/v1/users/checkins_controller_spec.rb

```diff
@@ -120,24 +121,25 @@
 
       it "fetches the user's last device checkin with all attributes" do
         get :last, params: params
-        expect(res_hash.first['id']).to be checkin.id
-        expect(res_hash.first.keys).to match checkin.attributes.keys
+        expect(res_hash[:checkins].first['id']).to be checkin.id
+        # TODO: define required set of attributes
+        # expect(res_hash[:checkins].first.keys).to match checkin.attributes.keys
```

Comment:
Ok understood, ideally we will have a way for this endpoint to return all check-in attributes if the request has the copo-app-key present but this is ok for now.